src/renderers/dom/shared/DOMProperty.js

```diff
@@ -16,6 +16,7 @@ var invariant = require('fbjs/lib/invariant');
 // These attributes should be all lowercase to allow for
 // case insensitive checks
 var RESERVED_PROPS = {
+  aria: true,
```

Comment:
Where is `data`?

We might not need to reserve these. A warning is probably enough.

Reply:
It turned out we actually use `data` and have it in whitelist. Apparently it's useful for `<object>`. Yes, could make it a warning.